CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Added
 
+- Added environmentId to BPOINT configuration
```

Comment:
The merge of this doesnt look right to me, missing the new release 0.7.1

Reply:
Yeh I havent merged yet. Ill do it now